src/dune_rules/foreign_rules.mli

```diff
@@ -13,7 +13,7 @@ val build_o_files
   -> foreign_sources:Foreign.Sources.t
   -> dir:Path.Build.t
   -> expander:Expander.t
-  -> requires:Lib.t list Resolve.t
+  -> requires:(Lib.t * bool) list Resolve.t
```

Comment:
This seems wrong: this function deals with building native sources, so I don't think the boolean is relevant here (am I missing something?). If that's the case, there is no reason to change anything here.